teams/cloud-compute.toml

```diff
@@ -9,16 +9,14 @@ leads = []
 members = [
     "albertlarsan68",
     "asquared31415",
-    "Dajamante",
     "jhpratt",
     "JoelMarcey",
     "nellshamrell",
-    "ouz-a",
     "Sp00ph",
     "yaahc",
-    "WaffleLapkin",
     "JulianKnodt",
     "vincenzopalazzo",
+    "hkBst",
 ]
```

Comment:
```suggestion
]
alumni = [
    "Dajamante",
]
```

Reply:
Not sure why it insists on the -] +] ...

Reply:
That's just the way the diff is rendered.

Reply:
Also add /workspaces/team/people/hkBst.toml with these contents:
name = "Marijn Schouten"
github = "hkBst"
github-id = 25009929
email = "mhkbst@gmail.com"
zulip-id = 878204